logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/Conversation.kt

```diff
@@ -22,9 +22,30 @@ data class Conversation(
     val access: List<Access>,
     val accessRole: List<AccessRole>?
 ) {
-    enum class Type { SELF, ONE_ON_ONE, GROUP, CONNECTION_PENDING }
-    enum class AccessRole { TEAM_MEMBER, NON_TEAM_MEMBER, GUEST, SERVICE }
-    enum class Access { PRIVATE, INVITE, LINK, CODE }
+    enum class Type {
+        SELF,
+        ONE_ON_ONE,
+        GROUP,
+        CONNECTION_PENDING;
+    }
+
+    enum class AccessRole {
+        TEAM_MEMBER,
+        NON_TEAM_MEMBER,
+        GUEST,
+        SERVICE;
+    }
+
+    enum class Access {
+        PRIVATE,
+        INVITE,
+        LINK,
+        CODE;
+    }
+
+    fun supportsUnreadMessageCount() =
+        type == Type.ONE_ON_ONE || type == Type.GROUP
```

Comment:
Nitpick. More idiomatic getter instead of function.
```suggestion
    val supportsUnreadMessageCount
        get() = type in setOf(Type.ONE_ON_ONE, Type.GROUP)
```